x-pack/plugins/files/server/routes/file_kind/update.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import type { FileJSON } from '../../../common/types';
+import type { FileKindsRequestHandler } from './types';
+import { findFile } from './helpers';
+
+export const method = 'patch' as const;
+
+export const bodySchema = schema.object({
+  name: schema.maybe(schema.string()),
+  alt: schema.maybe(schema.string()),
+  meta: schema.maybe(schema.object({}, { unknowns: 'allow' })),
```

Comment:
I like this idea. I don't think we need to enforce the `meta` check though. I can imagine it might be useful to have some nesting for apps. Let's think about it a little more!